### ISSUE TITLE
Continued training of the original BERT models (not to PyTorch)

### ISSUE THREAD

**Issue by singhsidhukuldeep** [huggingface/transformers#8756]
# 🚀 Feature request

I am looking for continued training of original tiny-bert with my own raw data using masked language modelling. 
But I want the final model in TF.

## Motivation

I tried this [LM](https://github.com/huggingface/transformers/blob/master/examples/language-modeling/run_mlm_wwm.py), but this only works from pytorch to pytorch.

I tried with using original ckpt and from_tf, it always results in a .h5 error 

Please let me know I can explain more OR help in any way.

Basically, we should be able to use tf weights >> masked language modelling >> and have a domain specific pre-trained Tensorflow language model.

**Comment by LysandreJik** [comment]
You should first convert your checkpoint to a huggingface checkpoint. You can check the docs on how to do that [here](https://huggingface.co/transformers/converting_tensorflow_models.html#bert).
